python/cuml/ensemble/randomforest.pyx

```diff
@@ -460,11 +473,12 @@ class RandomForestClassifier(Base):
         X : array-like (device or host) shape = (n_samples, n_features)
             Dense matrix (floats or doubles) of shape (n_samples, n_features).
             Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
-            ndarray, cuda array interface compliant array like CuPy
+            ndarray, __cuda_array_interface__ compliant array like CuPy
```

Comment:
I’d either change this acronym in all of the docstrings in every class or leave this one as is for consistency